pkg/cmd/config.go

```diff
@@ -502,6 +512,8 @@ name: {{ .ClusterName }}
 
 versions:
   kubernetes: "{{ .KubernetesVersion }}"
+kubeletConfiguration:
+ containerLogMaxSize: "{{ .ContainerLogMaxSize }}"
```

Comment:
Very tiny nit: I would put this at the end, so variables are grouped by priority/importance.